AlphaWallet/TokenScriptClient/Models/XMLHandler.swift

```diff
@@ -390,6 +390,7 @@ private class PrivateXMLHandler {
         }
         return Token(
                 id: tokenId,
+                tokenType: TokenType.erc875,
```

Comment:
@hboon slightly put off by this

Reply:
@James-Sangalli Good call.

How about adding a `tokenType: TokenType` arg to this function:

`func getToken(name: String, symbol: String, fromTokenId tokenId: TokenId, index: UInt16, inWallet account: Wallet, server: RPCServer, callForAssetAttributeCoordinator: CallForAssetAttributeCoordinator) -> Token {`

I noticed that callers either have a `TokenObject` available (which has a `tokenType` property) or should at least have the contract and TokensDataStore (or ServerDictionary<TokenDataStore> which you can use an RPCServer as the key to get the correct TokensDataStore) instance, and then the `TokenObject` to read its `tokenType` property from.

We might have to re-examine `Token` later to see if it's always used correctly or whether it's necessary at all.